logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc

```diff
@@ -7,6 +7,52 @@ include::modules/common-attributes.adoc[]
 toc::[]
 :toclevels: 2
 
+[id="status-of-elasticsearch-cluster"]
+== Status of Elasticsearch Cluster
+
+The Elasticsearch cluster uses the following fields to indicate its status.
```

Comment:
@igor-karpukhin Can we improve on this description of where and how the user can get the following messages?

Reply:
@rolfedh I'd mention the dashboard in a Monitoring section in Openshift Console (`<cluster_url>/monitoring/dashboards/grafana-dashboard-cluster-logging`)

Reply:
Updated thanks. @igor-karpukhin  Please review and approve this PR.